SofaKernel/modules/SofaBaseTopology/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ set(SOFABASETOPOLOGY_SRC "src/${PROJECT_NAME}")
 
 set(HEADER_FILES
     ${SOFABASETOPOLOGY_SRC}/config.h.in
+    ${SOFABASETOPOLOGY_SRC}/fwd.h
```

Comment:
I'm not very fan of this header name... especially if you have several fwd.h

Reply:
Me neither but as we have agreed for having multiple config.h I think it is better to be consistent.